concourse/scripts/create_concourse_cert.sh

```diff
@@ -1,16 +1,37 @@
 #!/bin/sh
 
-set -eu
+set -euo pipefail
 
 concourse_fqdn="${CONCOURSE_HOSTNAME}.${SYSTEM_DNS_ZONE_NAME}"
 
+get_route53_change_batch() {
+  action=${1}
+  cat <<EOF
+{
+  "Changes": [
+    {
+      "Action": "${action}",
+      "ResourceRecordSet": {
+        "Name": "${dns_validation_record}",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "${dns_validation_value}"
+          }
+        ]
+      }
+    }
+  ]
+}
+EOF
+}
+
 arn=$(aws acm list-certificates --query "CertificateSummaryList[?DomainName==\`${concourse_fqdn}\`].CertificateArn" --output text)
 
-created_cert="false"
 if [ -z "${arn}" ] || [ "${arn}" = "None" ]; then
   echo "Requesting certificate for ${concourse_fqdn}"
-  arn=$(aws acm request-certificate --domain-name "${concourse_fqdn}" --domain-validation-options "DomainName=${concourse_fqdn},ValidationDomain=${SYSTEM_DNS_ZONE_NAME}" --output text)
-  created_cert="true"
+  arn=$(aws acm request-certificate --domain-name "${concourse_fqdn}" --validation-method "DNS" --output text)
```

Comment:
gah I think the aws tools need updating in the container too!! 

Sorry I didn't catch this earlier! 

I currently get an error when running the pipeline:

```
Unknown options: --validation-method, DNS
```

Reply:
No, this is my fault. Sorry, that I didn't test this with the image. I'll fix it.

Reply:
PR for updating the AWS cli: https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/106